packages/fxa-auth-server/lib/server.js

```diff
@@ -381,9 +380,6 @@ async function create(
   );
   server.auth.strategy('oauthServerJWT', 'fxa-oauthServerJWT');
 
-  server.auth.scheme(authBearer.AUTH_SCHEME, authBearer.strategy);
-  server.auth.strategy(authBearer.AUTH_STRATEGY, authBearer.AUTH_SCHEME);
```

Comment:
this has been vestigial since oauth merged into auth-server... maybe even before then